src/status_im/components/toolbar/styles.cljs

```diff
@@ -1,103 +1,114 @@
 (ns status-im.components.toolbar.styles
-  (:require [status-im.components.styles :refer [text1-color
-                                                 color-white
-                                                 color-light-gray
-                                                 color-blue
-                                                 color-black]]))
+  (:require-macros [status-im.utils.styles :refer [defstyle defnstyle]])
+  (:require [status-im.components.styles :as styles]
+            [status-im.utils.platform :as p]))
 
-(def toolbar-background1 color-white)
-(def toolbar-background2 color-light-gray)
+(def toolbar-background1 styles/color-white)
 
 (def toolbar-height 56)
-(def toolbar-icon-width 32)
-(def toolbar-icon-spacing 8)
-
-(def toolbar-gradient
-  {:height 4})
-
-(defn toolbar-wrapper [background-color]
-  {:backgroundColor (or background-color toolbar-background1)
-   :elevation       2})
-
-(def toolbar
+(def toolbar-icon-width 24)
+(def toolbar-icon-height 24)
+(def toolbar-icon-spacing 24)
+
+(defnstyle toolbar [background-color flat?]
+  {:flex             0
+   :flex-direction   :row
+   :align-items      :center
+   :justify-content  :space-between
+   :background-color (or background-color toolbar-background1)
+   :elevation        (if flat? 0 2)
+   :android          {:height 55}
+   :ios              {:height 56}})
+
+(defnstyle toolbar-nav-actions-container
+  [actions]
   {:flex-direction :row
-   :height         toolbar-height})
-
-(defn toolbar-nav-actions-container [actions]
-  {:width          (when (and actions (> (count actions) 0))
-                     (-> (+ toolbar-icon-width toolbar-icon-spacing)
-                         (* (count actions))
-                         (+ toolbar-icon-spacing)))
-   :flex-direction "row"})
-
-(def toolbar-nav-action
-  {:width           toolbar-height
-   :height          toolbar-height
-   :align-items     :center
-   :justify-content :center
-   :padding-right   12})
+   :margin-left    4})
+
+(defstyle toolbar-container
+  {:flex       1
+   :android    {:padding-left 18}
+   :ios        {:align-items  :center}})
 
 (def toolbar-title-container
   {:flex           1
-   :alignItems     :center
-   :justifyContent :center})
+   :flex-direction :column
+   :margin-left    6})
+
+(defstyle toolbar-title-text
+  {:color          styles/text1-color
+   :letter-spacing -0.2
+   :font-size      17
+   :ios            {:text-align "center"}})
+
+(def toolbar-border-container
+  (get-in p/platform-specific [:component-styles :toolbar-border-container]))
```

Comment:
maybe good chance to get rid of `p/platform-specific`

Reply:
We have a specific issue for this. I'd rather keep that PR focused on `toolbar` migration, it's already quite big!

Reply:
ok, thanks